system/src/system_task.cpp

```diff
@@ -492,7 +494,11 @@ int system_isr_task_queue_free_memory(void *ptrToFree) {
 
 static void process_isr_task_queue()
 {
-    SystemISRTaskQueue.process();
+    // Process up to SYSTEM_ISR_TASK_QUEUE_MAX_TASKS_PER_ONE_PROCESS per one invocation
+    // This is simply to to avoid accidentally running into a situation where a processed task generates another task indefinitely
```

Comment:
Is this really necessary? I would expect such a recursive task to be executed on the next iteration without causing any problem.

Reply:
Refactored slightly as discussed.